feat(transformers): add tests (#4153)
algolia/instantsearch.js: scripts/transforms/README.md

## Codemods

> These codemods (code transformers) can be ran with `npx`. If you installed the `instantsearch.js` package already with `yarn`, you can ru`yarn instantsearch-scripts`).

Author: ```suggestion
These codemods (code transformers) can be ran with `npx`. If you installed the `instantsearch.js` package already with `yarn`, you can run them like this: `yarn instantsearch-scripts`).
```

> ### `addWidget-to-addWidgetss`

Reviewer: there is a typo here, no?

Author: the title is what it does, not the name of the transform

Author: ```suggestion
### `addWidget-to-addWidgets`
```

This will replace calls to `addWidget(widget)` to `addWidgets([widget])`.

```
npx instantsearch.js addWidget-to-addWidgets <path>
```

### Notes

> If you are using Prettier or eslint, make sure to run its autofixing after this transformation, since it's possible that code should be formatted differently after it has been transformed. For example in our repository using prettier that would be:

Reviewer 2: ```suggestion
If you are using Prettier or ESLint, make sure to run its autofixing after this transformation, since code can be formatted differently after it has been transformed. For example, in our repository, the Prettier command would be:
```
